test/tests/functional/pbs_qmgr.py

```diff
@@ -76,13 +76,17 @@ def test_listcmd_whitespaces(self):
         node_prefix = "vn"
         nodename = node_prefix + "[0]"
         vndef_file = None
+        qmgr_path = os.path.join(self.server.pbs_conf["PBS_EXEC"], "bin",
+                                 "qmgr")
+        self.assertTrue(os.path.isfile(qmgr_path))
```

Comment:
Don't assert here.  It will fail the test if qmgr isn't available.  Skip the test instead.